sonic_data_client/db_client.go

```diff
@@ -112,6 +113,14 @@ func NewDbClient(paths []*gnmipb.Path, prefix *gnmipb.Path) (Client, error) {
 	if UseRedisLocalTcpPort {
 		useRedisTcpClient()
 	}
+
+	// TODO: Remove debug log
+	for _, _path := range paths {
+		fmt.Printf("single path: %v\n", _path)
```

Comment:
Could you please revert all necessary change in this file? I know it may not be caused by you. Please help. It distracts this  review. Thanks!

Reply:
I think these are gofmt changes.